data/wanderer/wanderers start.txt

```diff
@@ -377,19 +321,6 @@ event "wanderer technology available" # Name unchanged for compatibility
 
 
 
-# Compatibility patch for saves before "language: Hai" variable was added.
-mission "Hai Language Patch"
```

Comment:
Are there other missions that depend on you having the "language: Hai" condition? Best to check.

Reply:
That'll be a single mission, and quite a short one, so I'll PR it directly to endless-sky master. It's just going to say that the ban on human-Hai translators has been lifted, so the translators that have been available for a century are allowed now. They're government-subsidized, so you'll be offered one for free if you land on any inhabited Hai planet or station.

Reply:
It is in my fork's `hai-translator-ban-lifted` branch, and I'll PR it at the same time as this branch.